lms/envs/common.py

```diff
@@ -1197,7 +1197,9 @@ def _make_mako_template_dirs(settings):
 # once the responsibility of XBlock creation is moved out of modulestore - cpennington
 XBLOCK_MIXINS = (LmsBlockMixin, InheritanceMixin, XModuleMixin, EditInfoMixin)
 
-# Allow any XBlock in the LMS
```

Comment:
Do you understand what this comment meant, and why it is ok to delete?  I'm not clear on what it meant.

Reply:
it's explaining what `prefer_xmodules` does, instead of the settings variable

Reply:
Would it be useful (and accurate?) to say something in the description like:
```
The default of prefer_xmodules allows any XBlock in the LMS, prefering xmodules over XBlocks when multiple matches are found.
```